### PR TITLE
[CARBONDATA-518]Fix the bug of CarbonExample in spark1.5 moudle

### PR DESCRIPTION
**Pull request by Zhangshunyu**
## Why raise this pr?
CarbonExample of spark moudle can not run as kettlehome and storepath shoud get form carbon properties now
## How to test?
Pass CI

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/85/

File: examples/spark/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala [hunk 1]
```diff
@@ -26,11 +28,15 @@ object CarbonExample {
   def main(args: Array[String]) {
     val cc = ExampleUtils.createCarbonContext("CarbonExample")
     val testData = ExampleUtils.currentPath + "/src/main/resources/data.csv"
-
+    val rootPath = new File(this.getClass.getResource("/").getPath
```
Comment (Zhangshunyu) [inline examples/spark/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala]: @chenliang613 Hi liang, now the path is not get from carboncontext, they are got from carbonproperties, the code has been changed by this commit:
https://github.com/apache/incubator-carbondata/commit/f67ec0125247e98d7925866fd3062ee3ddf16e28https://github.com/apache/incubator-carbondata/commit/f67ec0125247e98d7925866fd3062ee3ddf16e28 
And you can try carbon example, its not run now.

**Comment by Zhangshunyu** [comment]
@chenliang613 Fixed, pls check.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/100/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**